UefiPayloadPkg/SPI/pci_mmio_cfg.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PCI_MMIO_CFG_H
```

Comment:
Use PciLib from EDK2 and delete this file

Reply:
deleted